### DISCUSSION TITLE
Python Version Set Incorrectly

### DISCUSSION THREAD

**Discussion by mgrover1** [ARM-DOE/pyart#1398] [Q&A]
This question was raised in a pull request

https://github.com/ARM-DOE/pyart/pull/1397#issuecomment-1462162872

@wxwolff how are you are installing Py-ART here?
- conda-forge (`conda install -c conda-forge arm_pyart`)
- pip (`pip install arm_pyart`)
- from source (`pip install -e .`)

**Comment by wxwolff** [comment]
Conda



____________________________________________
David B. Wolff, Code 612
Assistant Chief
Mesoscale Atmospheric Processes Lab
GPM Deputy Project Scientist for GV
GPM GV System Manager
NASA Wallops Flight Facility
Wallops Island, VA 23337
Ph: 757-824-1515; Cell: 443-262-1146



From: Max Grover ***@***.***>
Reply-To: ARM-DOE/pyart ***@***.***>
Date: Thursday, March 9, 2023 at 09:39 AM
To: ARM-DOE/pyart ***@***.***>
Cc: "Wolff, David B. (WFF-6120)" ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] [ARM-DOE/pyart] Python Version Set Incorrectly (Discussion #1398)


This question was raised in a pull request

#1397 (comment)<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FARM-DOE%2Fpyart%2Fpull%2F1397%23issuecomment-1462162872&data=05%7C01%7CDavid.B.Wolff%40nasa.gov%7C562f617cb64c4fc759ad08db20ac0970%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C638139695511759724%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=RL12670T%2FFW%2FDgETIZMQ%2B8QHSwAaSx1dPPLeRlWwDOQ%3D&reserved=0>

@wxwolff<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fwxwolff&data=05%7C01%7CDavid.B.Wolff%40nasa.gov%7C562f617cb64c4fc759ad08db20ac0970%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C638139695511759724%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=aRszXmdA563xN3pB0kjq7iwTOvleO50QQ2LW6SAq%2F%2FE%3D&reserved=0> how are you are installing Py-ART here?

  *   conda-forge (conda install -c conda-forge arm_pyart)
  *   pip (pip install arm_pyart)
  *   from source (pip install -e .)

**Reply by mgrover1** [reply]
Thanks for raising this issue - we should have a new release on conda later today with the fix here. We needed to add our versioning tool to required packages for conda.

https://github.com/conda-forge/arm_pyart-feedstock/pull/79

**Reply by wxwolff** [reply]
Thanks.

**Reply by zssherman** [reply]
@wxwolff The new release is on conda-forge now. Let us now if the issue has been resolved!

**Reply by mgrover1** [reply]
@wxwolff when you get a chance can you update your Py-ART version and test to see if this has been resolved? I tested it on my machine this morning, and it is working as expected, but I wanted to make sure it works for you as well.

```bash
conda update arm_pyart
```

**Comment by wxwolff** [comment]
Yes, it works! After update…

>>>pyart.__version__
>>> ‘1.14.1.post1’

Dave


__________________________________________________
David B. Wolff, Code 612
Assistant Chief
Mesoscale Atmospheric Processes Lab
GPM Deputy Project Scientist for GV
GPM GV System Manager
NASA Wallops Flight Facility
Wallops Island, VA 23337
Ph: 757-824-1515; Cell: 443-262-1146


From: Max Grover ***@***.***>
Reply-To: ARM-DOE/pyart ***@***.***>
Date: Friday, March 10, 2023 at 1:35 PM
To: ARM-DOE/pyart ***@***.***>
Cc: "Wolff, David B. (WFF-6120)" ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [ARM-DOE/pyart] Python Version Set Incorrectly (Discussion #1398)


@wxwolff<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fwxwolff&data=05%7C01%7CDavid.B.Wolff%40nasa.gov%7C54d9fe7d830641b1d2c908db219638cc%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C638140701304187753%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BRaCZYwmrhIwonLBkrGXWyE%2B5flPiu1iMiAt%2FG3aBkE%3D&reserved=0> when you get a chance can you update your Py-ART version and test to see if this has been resolved? I tested it on my machine this morning, and it is working as expected, but I wanted to make sure it works for you as well.

conda update arm_pyart
